util/build.go

```diff
@@ -22,7 +22,6 @@ import "runtime"
 var (
 	// These variables are initialized via the linker -X flag in the
 	// top-level Makefile when compiling release binaries.
-	buildVers string // Go Version
```

Comment:
Should we be populating this at build time. Or is it sufficient to get this information from [runtime.Version()](https://golang.org/pkg/runtime/#Version)?